tests/unit/toContainChunk/index.js

```diff
@@ -1,4 +1,4 @@
-import diff from 'jest-diff';
+import { diff } from 'jest-diff';
```

Comment:
Not strictly related to this PR, but as I was running the tests, my new test initially failed, and I had to change this to make the test error correctly.